dist.ini

```diff
@@ -0,0 +1,24 @@
+abstract = Cryptographically secure v4 UUIDs for Linux x64
+name     = UUID4-Tiny
+author   = CV-Library <cvlibrary@cpan.org>
+license  = Perl_5
+copyright_holder = CV-Library
+copyright_year   = 2019
+
+[AutoPrereqs]
+[GitHub::Meta]
+[GatherDir]
+    exclude_filename = dist.ini
+[License]
+[MetaJSON]
+[MetaYAML]
+[MetaProvides::Package]
+[ReadmeAnyFromPod / MarkdownInBuild]
+filename = README.md
+[CopyFilesFromBuild]
+copy = README.md
+[VersionFromModule]
+; Manifest should go as close to last as possible, according to docs
+[Manifest]
+; PrereqsClean should go last, according to docs
+[PrereqsClean]
```

Comment:
Not sure what we are using this for, but in my machine (with Perl 5.30 and v.0.93 of the distribution) it's generating this warning:

```
Smartmatch is experimental at .../Dist/Zilla/Plugin/PrereqsClean.pm line 186, <GEN1> line 24.
```

Reply:
Removed as it doesn't seem to change anything on this repo anyway.